code/modules/goonchat/browserOutput.dm

```diff
@@ -189,14 +189,17 @@ GLOBAL_DATUM_INIT(iconCache, /savefile, new("data/iconCache.sav")) //Cache of ic
 	var/list/partial = splittext(iconData, "{")
 	return replacetext(copytext(partial[2], 3, -5), "\n", "")
 
+GLOBAL_LIST_EMPTY(bicon_cache)

```

Comment:
make the bicon_cache an empty global list

Reply:
dont make global lists in random files. put them in the right place

Reply:
which is what place? this is just how goonchat had it, and it worked, /tg/ copied it incorrectly

Reply:
so in __HELPERS/unsorted.dm?

Reply:
theres a folder for global lists somewhere